people-and-planet-ai/land-cover-classification/data/prediction-locations.csv

```diff
@@ -0,0 +1,37 @@
+name,lat,lon,year
```

Comment:
Shouldn't data should go in gs://cloud-samples-data/vertex-ai/?

Reply:
We've stored other CSV files in GitHub for other samples before. They're small enough to not have a performance impact, plus it gives more visibility for people on what's on the file or if they want to edit them. It also makes it simpler to run locally.

I prefer it like this, but I can upload them to Cloud Storage if you have a strong opinion about it.

Reply:
I'm pro keeping it in github if it has no negative performance impact because it's easier maintenance wise